src/ConcreteOperations/volume.jl

```diff
@@ -0,0 +1,34 @@
+export volume
+
+"""
+    volume(H::AbstractHyperrectangle)
+
+Return the volume of a hyperrectangular set.
+
+### Input
+
+- `H` -- hyperrectangular set
+
+### Output
+
+The (Euclidean) volume of ``H``.
```

Comment:
Just out of curiosity: What other types of volume are there?

Reply:
I meant the volume element in [Euclidean space](https://en.wikipedia.org/wiki/Volume_element#Volume_element_in_Euclidean_space). This function estimates the [Lebesgue measure](https://en.wikipedia.org/wiki/Lebesgue_measure) of the set, but there are many other ways to assign volumes by using a different measure.